http/src/main/resources/swagger/api-docs.yaml

```diff
@@ -78,6 +80,60 @@ paths:
                 $ref: "#/components/schemas/LeonardoVersion"
       security: []
 
+  ## Appv2 Paths ##
+
+  "/api/apps/v2/{workspaceId}":
```

Comment:
Should we add the other routes (getApp, createApp, deleteApp) to swagger as well?

Reply:
Added!